icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/repository/CategoryRepository.java

```diff
@@ -1,7 +1,9 @@
 package com.summerschool.icecreamshop.repository;
 
 import com.summerschool.icecreamshop.model.Category;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
-public interface CategoryRepository extends CrudRepository<Integer, Category> {
+@Repository
+public interface CategoryRepository extends JpaRepository<Category, Integer> {
```

Comment:
Please make sure that the ID parameter has the same type as the ID property inside your Category Entity, meaning that you should have Long instead of Integer.